src/Microsoft.DotNet.XHarness.CLI/iOS/iOSTestCommandArguments.cs

```diff
@@ -4,36 +4,96 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using Microsoft.DotNet.XHarness.CLI.Common;
+using Microsoft.DotNet.XHarness.iOS.Shared;
+using Microsoft.DotNet.XHarness.iOS.Shared.Utilities;
 
 namespace Microsoft.DotNet.XHarness.CLI.iOS
 {
     internal class iOSTestCommandArguments : TestCommandArguments
     {
+        /// <summary>
+        /// Path to where Xcode is located.
+        /// </summary>
+        public string XcodeRoot { get; set; }

```

Comment:
We should add the same we added in xamarin-macions, a lazy version, if the path is not provided, use xcode-select, but I'll create an issue for this and we can move fwd.

Reply:
This already happens inside `ProcessManager` I tihnk. Should we move it out of there?

Reply:
yes, I created an issue so that we do not forget: https://github.com/dotnet/xharness/issues/49

Feel free to state it there, then I think we only need to change the mono options to use ':'